Dalamud/Interface/ImGuiBackend/Renderers/Dx11Renderer.cs

```diff
@@ -264,7 +264,7 @@ private void RenderDrawDataInternal(
             this.indexBuffer.Dispose();
         if (this.indexBuffer.Get() is null)
         {
-            this.indexBufferSize = drawData.TotalIdxCount + 5000;
+            this.indexBufferSize = drawData.TotalIdxCount + 16384;
```

Comment:
Where do these numbers come from?

Reply:
There's no justification for this really but since it's 32KB of more VRAM usage I thought, why not.